pysheaf/pysheaf.py

```diff
@@ -795,12 +827,18 @@ def consistencyRadius(self,assignment,testSupport=None,tol=1e-5):
                                             if rad > radius:
                                                 count_comparison+=1
                                                 radius = rad
-
         if count_comparison == 0:
-            warnings.warn("No SectionCells in the assignment match, therefore nothing was compared by consistencyRadius")
-        
+            warnings.warn("No SectionCells in the assignments match, therefore nothing was compared by assignmentMetric")
```

Comment:
Um, the error message appears to be wrong here?

Reply:
The error message is definitely wrong. I will cancel this pull request, update, and submit a new request.